packages/relative-values/src/components/View/Header.tsx

```diff
@@ -6,23 +6,20 @@ import { CellBox } from "./CellBox";
 
 export const Header: FC<{
   item: Item;
-  clickable?: boolean;
-}> = ({ item, clickable }) => {
+}> = ({ item }) => {
   const {
     catalog: { clusters },
   } = useSelectedInterface();
 
   const cluster = item.clusterId ? clusters[item.clusterId] : undefined;
   return (
-    <CellBox header clickable={clickable}>
```

Comment:
I removed the <Cellbox> component from this element, to make this element more flexible.